diff --git a/dist/server/tsdoc/base.js b/dist/server/tsdoc/base.js
index abd7256e30435464b3db069c2965600a4f8156c4..3d5dbeeec818ba5d527d5a6af8e9cb297ead7fe9 100644
--- a/dist/server/tsdoc/base.js
+++ b/dist/server/tsdoc/base.js
@@ -4,7 +4,7 @@ import { Project, SyntaxKind, ts } from "ts-morph";
 import { CWD } from "../constants.js";
 import { logger } from "../utils.js";
 const project = new Project({
-  tsConfigFilePath: "./tsconfig.json",
+  tsConfigFilePath: path.join(CWD, "tsconfig.json"),
   skipAddingFilesFromTsConfig: true,
   compilerOptions: {
     // Do not show `undefined` value in type for optional value
