diff --git a/dist/server/loader.js b/dist/server/loader.js
index bd749814ff1c5b73bc26cde7bb02fb2af76530dc..f50db71979eabfde455db0ad9634afed81469282 100644
--- a/dist/server/loader.js
+++ b/dist/server/loader.js
@@ -92,7 +92,7 @@ async function loader(source) {
     );
   }
   const lastCommitTime = IS_PRODUCTION ? await getLastCommitTime(resourcePath) : NOW;
-  if (!IS_PRODUCTION && resourceQuery === METADATA_ONLY_RQ) {
+  if (resourceQuery === METADATA_ONLY_RQ) {
     return compileMetadata(source, {
       filePath: resourcePath,
       lastCommitTime
