diff --git a/dist/server/tsdoc/base.js b/dist/server/tsdoc/base.js
index 5d9a6dc495a25aa634cd4c190979f76fe46e4ea0..956dc1a5e375455add37f69d5eb52e3a02468694 100644
--- a/dist/server/tsdoc/base.js
+++ b/dist/server/tsdoc/base.js
@@ -14,12 +14,13 @@ const project = new Project({
   }
 });
 const DEFAULT_FILENAME = "$.ts";
-const { compilerObject } = project.getTypeChecker();
+let compilerOptions;
 function generateDefinition({
   code,
   exportName = "default",
   flattened = false
 }) {
+  compilerOptions ||= project.getTypeChecker().compilerObject;
   const sourceFile = project.createSourceFile(DEFAULT_FILENAME, code, {
     overwrite: true
   });
